src/nbl/ui/CWindowWin32.cpp

```diff
@@ -16,11 +15,11 @@ namespace ui
 	};
 	struct SCreateWindowRequest : SRequest
 	{
-		SCreateWindowRequest(int _x, int_y, uint32_t _w, uint32_t _h, CWindowWin32::E_CREATE_FLAGS _flags, CWindowWin32::native_handle_t wnd) : 
+		SCreateWindowRequest(int32_t _x, int32_t _y, uint32_t _w, uint32_t _h, CWindowWin32::E_CREATE_FLAGS _flags, CWindowWin32::native_handle_t wnd) :
```

Comment:
see IWindow::SCreationParams, use this as input for this request (you may add members to it)

Reply:
Nah native window handle is the only thing that doesn't allow me to do it

Reply:
erm actually why do you have window handle as an input for window creation request?

Reply:
Well, as a return parameter.

Reply:
yeah but its pass-by-copy here

Reply:
yep, my bad